codegen/src/main/java/io/sundr/codegen/functions/ClassTo.java

```diff
@@ -261,11 +261,12 @@ private static void processAnnotatedElement(AnnotatedElement field, List<Annotat
       AnnotationRef annotationRef = ANNOTATIONTYPEREF.apply(annotationType);
       Map<String, Object> parameters = new HashMap<>();
       for (java.lang.reflect.Method method : annotationType.getDeclaredMethods()) {
+        final String name = method.getName();
         try {
           final Object value = method.invoke(annotation, (Object[]) null);
-          parameters.put(method.getName(), value);
+          parameters.put(name, value);
         } catch (IllegalAccessException | InvocationTargetException e) {
-          throw new RuntimeException(e);
+          System.out.printf("Couldn't retrieve '%s' parameter value for %s%n", name, annotationType.getName());
```

Comment:
Can we address the `IllegalAccessException` in the first place?

Reply:
If you know how to do it, sure… :)